cinema/views.py

```diff
@@ -98,6 +96,29 @@ def get_queryset(self):
 
         return queryset.distinct()
 
+    @extend_schema(
+        parameters=[
+            OpenApiParameter(
+                name="title",
+                type=OpenApiTypes.STR,
+                location=OpenApiParameter.QUERY,
+                description="filtering by title",
+            ),
+            OpenApiParameter(
+                "genres",
+                type=OpenApiTypes.ANY,
```

Comment:
Not true why it accept Any. Please always use a specific type instead of Any!!!!